app/views/layouts/application.html.slim

```diff
@@ -15,3 +15,4 @@ html
     - unless controller_name.in? ['sessions', 'registrations', 'passwords', 'calculators', 'contact_us', 'messages']
       = render 'layouts/homepage'
     = yield
+    = any_login_here if defined?(AnyLogin)
```

Comment:
where is configuration by roles for user?)

Reply:
> where is configuration by roles for user?)

updated, please check)